### ISSUE TITLE
comment about `[Editorial]`

### ISSUE THREAD

**Issue by ousia** [standardebooks/manual#161]
I have just spotted in Jowett’s Plato both:

> _Archiv fur Geschichte der Philosophie_

and

> _an und fur sich seyn_

In German, it is _für_ and this is wrong, not matter what the typesetter or the editor may have thought.

I see that [§ 1.4.2](https://standardebooks.org/manual/1.8.0/single-page#1.4.2) requires to mark such a basic spelling mistake (in a foreign language) as `[Editorial]`.

Sorry but I think it is totally different to modernize spelling (which would be _an und für sich sein_ in that case) than correcting spelling mistakes (probably from the typesetter).

My comment is that it makes it harder to report a such a basic error, since it adds the task to check the text in the printed edition.

Results after searching the scanned books from the Internet Archive:
 
The first error is in the digital transcription, [not included in the printed edition](https://archive.org/details/in.ernet.dli.2015.8098/page/n35/mode/2up?view=theater).

The second mistake may be from the typesetter, since [it is included in the printed edition](https://archive.org/details/dli.ministry.11928/page/419/mode/2up?view=theater).

This would require two commits for fixing the same word to the same result: one with `[Editorial]` and another one without it.

Sorry, but I think it may be worth to rule also these cases.

Many thanks for your help.

**Comment by acabal** [comment]
Yes, you can fix those as you suggest. Fixing two characters and writing two brief commit messages is not hard, you could have done that and opened a PR in the time it took you to compose this issue :)

**Comment by ousia** [comment]
No, checking each single passage is the time-consuming task.

I don’t know why I’m not given access to my own repositories.

I will try tomorrow. I would like to use the web interface for commits.

Many thanks for your help.

**Comment by ousia** [comment]
Closing since `[Editorial]` is taken literally (so no further discussion).
